src/js/popup/reducers/spentTime.js

```diff
@@ -0,0 +1,74 @@
+import { ON_USER_ID_LOAD, ON_NOTES_LOAD, ON_NOTES_PENDING, STOP_TIMER } from '../constants/ActionTypes';
+
+const initialState = {
+  userId: -1,
+  time: 0
+};
+
+function sum(x, y) {
+  return x + y;
+}
+
+function parseTimeElement(element) {
```

Comment:
I think we can extract parsing logic to a separate module and place it in a new `services` folder.